src/transforms/corrcholesky.jl

```diff
@@ -30,9 +30,9 @@ function TV.transform_with(flag::TV.LogJacFlag, t::CorrCholesky, x::AbstractVect
     return Cholesky(U, 'U', 0), ℓ, index
 end
 
-TV.inverse_eltype(t::CorrCholesky, x::AbstractMatrix) = TV.extended_eltype(x)
+TV.inverse_eltype(t::CorrCholesky, x::AbstractMatrix) = eltype(x)
 
-TV.inverse_eltype(t::CorrCholesky, x::Cholesky) = TV.extended_eltype(x)
+TV.inverse_eltype(t::CorrCholesky, x::Cholesky) = eltype(x)
```

Comment:
Within the source, we use `TV` as shorthand for `TransformVariables`. I think all the TV stuff can soon be moved to an extension, since the new transport code in MeasureBase can subsume this capability. 

TV used to have an `extended_eltype` function, but now that throws an error. I don't know details of how this was different from `eltype`, I was just mimicking the code style from that package.